jarbas/api/views.py

```diff
@@ -26,9 +26,16 @@ def get(self, request):
         values = map(self.request.query_params.get, params)
         filters = {k: v for k, v in zip(params, values) if v}
 
-        # filter queryset
-        if self.request.query_params.get('suspicions'):
+        # filter suspicions
+        if self._bool_param('suspicions'):
             self.queryset = self.queryset.suspicions()
+
+        # filter reimbursement in latest dataset
+        in_latest = self._bool_param('in_latest_dataset')
+        if in_latest is not None:
```

Comment:
* `None`: user haven't user this filter
* `False`: user have used this filtering explicitly requesting records where `available_in_latest_dataset` is `False` (i.e. reimbursements deleted in the latest version of the Chamber of Deputies dataset)
* `False`: user have used this filtering explicitly requesting records where `available_in_latest_dataset` is `True` (i.e. reimbursements that are still in the latest version of the Chamber of Deputies dataset)